src/helm/benchmark/scenarios/contactcenter_convo_summarization_scenario.py

```diff
@@ -0,0 +1,124 @@
+"""
+Scenario for contact center conversational summarization.
+
+Loads input conversations defined in the ContactCenterConversationScenario
+and packs task specific input/output format for summarization tasks.
+
+Task structure
+
+    Conversation:
+            agent: message1
+            visitor: message2
+            ....
+    Summary:
+            summary of the conversation
+
+
+Example from the dataset
+    Conversation: 
+            agent: hi how can i help you today
+            visitor: i need help with my account
+            agent: sure what is your account number
+            visitor: 123456
+            .....
+    Summary:
+           - agent helped visitor with account number 123456
+"""
+
+
+import json
+import os
+from typing import List, Optional
+from helm.benchmark.scenarios.contactcenter_convo_base_scenario import ContactCenterConversationScenario
+from helm.common.general import ensure_file_downloaded, ensure_directory_exists
+from .scenario import Instance, Reference, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+
+class ContactCenterConversationSummarizationScenario(ContactCenterConversationScenario):
+    """
+    Scenario for contact center conversational summarization.
+    """
+
+    name = "cc_convo_summarization"
+    description = "Scenario for contact centern summarization tasks"
+    tags = ["cc_conversation_summarization"]
+
+    def __init__(
+        self,
+        dataset_path: str,
+        sampling_min_length: Optional[int] = None,
+        sampling_max_length: Optional[int] = None,
+        doc_max_length: Optional[int] = None,
```

Comment:
suggestion: names like
`doc_filter_min_length`, `doc_filter_max_length`, `doc_truncation_max_length`
or
`filter_doc_min_length`, `filter_doc_max_length`, `truncate_doc_max_length`
or
`doc_filter_min_words`, `doc_filter_max_words`, `doc_truncation_max_words`
would be more self-evident.